packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -517,7 +517,14 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
 
       globals.printError('Encountered error while creating the IPA:');
       globals.printError(errorMessage.toString());
-      globals.printError('Try distributing the app in Xcode: "open $absoluteArchivePath"');
+
+      final FileSystemEntityType type = globals.fs.typeSync(absoluteArchivePath);
+      globals.printError('Try distributing the app in Xcode:');
+      if (type == FileSystemEntityType.notFound) {
+        globals.printError('  open ios/Runner.xcworkspace');
```

Comment:
minuscule nit:  the old error msg is `Try distributing the app in Xcode: "open $absoluteArchivePath"`, but the new one would be `Try distributing the app in Xcode:  open $absoluteArchivePath`
not sure if thats intended?

Reply:
I think the quotes in the old one are because it's on the same line, so the quotes delineate the instructions vs the command to run.  The new one will print it on a newline with indentation, which matches a few other spots, (though the tool is pretty inconsistent):
https://github.com/flutter/flutter/blob/8e6643c597b54e5b802594fe73fd2132eb92f495/packages/flutter_tools/lib/src/ios/mac.dart#L203-L204
https://github.com/flutter/flutter/blob/8e6643c597b54e5b802594fe73fd2132eb92f495/packages/flutter_tools/lib/src/ios/devices.dart#L424-L425

Though, come to think of it, the `logger` has an `indent` parameter I can use.  I'll update.